packages/celo-typescript/tsconfig.json

```diff
@@ -0,0 +1,25 @@
+{
+  "compilerOptions": {
+    "composite": true,
+    "declaration": true,
+    "downlevelIteration": true,
+    "esModuleInterop": true,
+    "forceConsistentCasingInFileNames": true,
+    "jsx": "react",
+    "lib": ["DOM",  "DOM.Iterable", "es2015", "es2016"],
+    "module": "commonjs",
+    "moduleResolution": "node",
+    "noImplicitAny": true,
+    "noImplicitOverride": true,
```

Comment:
This was added on 4.3 - I'm curious if we have any projects using ts before that version, and if so, what would happen with this config?

Reply:
Nothing comes to mind that's on <4.3